### PR TITLE
Update README with better guides/instructions for contributing

### PR DESCRIPTION
**Pull request by kylekirkby**
Update README with better guides/instructions for contributing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,122 @@
...
+![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0)
 
-![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0) 
+The 96Boards website is built using the [Jekyll](https://jekyllrb.com/) static site generator. It uses the Linaro [jumbo-jekyll-theme](https://github.com/linaro-marketing/jumbo-jekyll-theme) (which is used for all of the Linaro static websites). With the source for the source for the 96Boards.org website available on GitHub in this repo, it makes it easy for community/technical contributors to submit content. The 96Boards documentation repo is synced into the _documentation directory during site builds that occur on our Atlassian Bamboo CI instance.
+
+## Contributions
+
+We happy to consider any contributions/feature requests that you may have. Please submit a PR with your changes and we will take a look. You can also use the `Github Edit` buttons available on all the website pages to help locate the file you wish to edit/raise and issue about.
```
Comment (sgermanserrano) [inline README.md]: Typos
```suggestion
We are happy to consider any contributions/feature requests that you may have. Please submit a PR with your changes and we will take a look. You can also use the `Github Edit` buttons available on all the website pages to help locate the file you wish to edit/raise an issue about.
```

File: README.md [hunk 2]
```diff
@@ -1,70 +1,122 @@
...
+![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0)
 
-![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0) 
+The 96Boards website is built using the [Jekyll](https://jekyllrb.com/) static site generator. It uses the Linaro [jumbo-jekyll-theme](https://github.com/linaro-marketing/jumbo-jekyll-theme) (which is used for all of the Linaro static websites). With the source for the source for the 96Boards.org website available on GitHub in this repo, it makes it easy for community/technical contributors to submit content. The 96Boards documentation repo is synced into the _documentation directory during site builds that occur on our Atlassian Bamboo CI instance.
```
Comment (pcolmer) [inline README.md]: "With the source for the source for" ... ?

Comment (pcolmer) [inline README.md]: "The 96Boards documentation repo is synced into the _documentation directory during site builds that occur on our Atlassian Bamboo CI instance." ... forget to turn off italics?

File: README.md [hunk 3]
```diff
@@ -93,8 +145,8 @@ keywords: |-
 # Product Short Description -  this is used on the product index page to provide a brief
 # overview of your product.
 product_short_desc: "Bitmain Sophon™ Edge Developer Board is powered by the BM1880"
-# Product Specification - This is the 96Boards specification of your product and is used 
-# by Jekyll when collectin products of a certain specification. 
+# Product Specification - This is the 96Boards specification of your product and is used
+# by Jekyll when collectin products of a certain specification.
```
Comment (sgermanserrano) [inline README.md]: Typo
```suggestion
# by Jekyll when collecting products of a certain specification.
```

File: README.md [hunk 4]
```diff
@@ -1,70 +1,122 @@
...
+![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0)
 
-![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0) 
+The 96Boards website is built using the [Jekyll](https://jekyllrb.com/) static site generator. It uses the Linaro [jumbo-jekyll-theme](https://github.com/linaro-marketing/jumbo-jekyll-theme) (which is used for all of the Linaro static websites). With the source for the source for the 96Boards.org website available on GitHub in this repo, it makes it easy for community/technical contributors to submit content. The 96Boards documentation repo is synced into the _documentation directory during site builds that occur on our Atlassian Bamboo CI instance.
+
+## Contributions
+
+We happy to consider any contributions/feature requests that you may have. Please submit a PR with your changes and we will take a look. You can also use the `Github Edit` buttons available on all the website pages to help locate the file you wish to edit/raise and issue about.
+
+### Simple edits on GitHub
```
Comment (pcolmer) [inline README.md]: This only works if you've got contributor permissions on the repository. Everyone else would need to fork the repo,

File: README.md [hunk 5]
```diff
@@ -1,70 +1,122 @@
...
+
+Once you've made your change you'll be able to add a change title and description. Make sure to select the "Create a new branch for this commit and start a pull request" option.
+
+![Simple GitHub changes - change text and description](/assets/images/content/create_new_branch.png)
+
+Then click "Propose file change" button. Maintainers of this website repo will then review and publish your change accordingly.
+
+If you'd like a more detailed guide on making changes on GitHub then view the GitHub documentation [here](https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository).
+
+### Using a the git CLI
```
Comment (pcolmer) [inline README.md]: "a the"

File: README.md [hunk 6]
```diff
@@ -1,70 +1,122 @@
...
+
+If you're familiar with using the git cli then you can simply fork this repository and clone your fork. Then submit pull request on a `new` branch based off of the `develop` branch. This is due to the 96Boards.org website repo following the [Git flow](https://danielkummer.github.io/git-flow-cheatsheet/) methodology. Push your changes to your fork and then create a pull request via the GitHub UI.
```
Comment (pcolmer) [inline README.md]: Consistency in capitalisation of CLI ...

Comment (pcolmer) [inline README.md]: "you can simply fork this repository and clone your fork" ... fork and clone? Surely just fork?

Comment (kylekirkby) [reply README.md]: If you're using the Git CLI then how will the repo be available to you by simply forking the repo?

File: README.md (hunk 4 above)
Comment (kylekirkby) [reply README.md]: Good spot! I've updated accordingly.

Comment (pcolmer) [reply README.md]: What does the GitHub web interface look like if you are not a contributor and you are making a simple edit? If it forces you to create a fork first and then a pull request, that's fine but you may want to update the screenshots if possible. You can then remove the comment about contributor permissions in the previous paragraph.

**Pull request merged**